### DISCUSSION TITLE
加载预训练模型为其他模型的操作

### DISCUSSION THREAD

**Discussion by aboy2018** [STVIR/pysot#352] [General] [answered]
我想以alexnet为baseline训练，但是加载的预训练模型为开源的alexnet最好的模型，此模型已经删掉了alexnet中的一些层，加载yaml文件时需要如何操作，求指教

**Comment by ZhiyuanChen** [comment, marked as answer]
Just like https://github.com/STVIR/pysot/blob/master/experiments/siamrpn_alex_dwxcorr/config.yaml
